Allows giving a custom name to the Survey results page

MaiklT opened the pull request:
## Changes made
Added keys to Resource files for control titles and module help, added messages to controls to display a link to the documentation

## PR Template Checklist

- [x] Fixes Bug
- [ ] Feature solution
- [ ] Other


## Please mark which issue is solved
Close #63

Commit:
Added the ability to change the title of the Results control. Closes #63

Commit:
Closes #63
Added ControlTitle keys and ModuleHelp

valadas commented on DNN.Survey/App_LocalResources/SurveyResults.ascx.resx:
```diff
@@ -117,9 +117,15 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="ControlTitle_surveyresults.text" xml:space="preserve">
+    <value>Ergebnisse</value>
```
This would should be english

MaiklT replied:
I changed it and commited it directly to the pull request.

valadas reviewed (changes requested):
Thanks, just one piece of text is not english...

MaiklT committed:
Update SurveyResults.ascx.resx

Corrected a value in the resource file that was in German for some reason (in fact to test ist, but forgot to turn it back to English...)

valadas reviewed (approved):
Awesome, thanks!
